zio/src/test/scala/ConsoleLogWriterTest.scala

```diff
@@ -231,7 +235,7 @@ final class ConsoleLogWriterTest extends AnyWordSpecLike with Matchers with zio.
     "created with level Trace" should {
       "log all the messages" in {
         val out = capturedConsoleOutOf {
-          val cl = consoleLogUpToLevel(Trace)
+          val cl = consoleLogUpToLevel(log.effect.LogLevels.Trace)
```

Comment:
for the record, no idea why this was needed, but not in all other cases...

Reply:
It's ambiguous with zio.Trace I guess.

Reply:
Yeah, you're absolutely right.. I'll create a mini-PR to hide it

Reply:
#530